webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BugIcon,
+  SecurityIcon,
+  EnhancementIcon,
+} from '@patternfly/react-icons';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+const ContentViewVersionErrata = ({ cvId, versionId, errataCounts }) => {
+  const {
+    total,
+  } = errataCounts;
+
+  const errataIcons = {
+    security: SecurityIcon,
+    bugfix: BugIcon,
+    enhancement: EnhancementIcon,
+  };
+
+  const getHref = type => `/content_views/${cvId}/versions/${versionId}/errata?queryPagedSearch=%20type%20%3D%20${type}`;
+
+  const ErrataLinkwithIcon = () => Object.keys(errataIcons).map((type) => {
+    const ErrataIcon = errataIcons[type];
+    return (
+      <React.Fragment key={type}>
+        <ErrataIcon title={type} />
+        <a href={getHref(type)}> {` ${errataCounts[type] || '0'}`} </a>
```

Comment:
This leading space makes the underline look funny when you hover over it. I'd prefer adding a `margin-left` for spacing if you need it.
```suggestion
        <a href={getHref(type)}> {`${errataCounts[type] || '0'}`} </a>
```

Reply:
I did drop the comma in between the Icons cause it would add some complexity to the simple map function we have here..Do you think we should go for the comma?
![Screenshot from 2021-04-27 15-30-28](https://user-images.githubusercontent.com/21146741/116300970-846de300-a76d-11eb-8473-07ae51658b86.png)

Reply:
Here's the mockup version with commas.
https://marvelapp.com/prototype/1121a8de/screen/72835873

![Screenshot from 2021-04-27 15-32-36](https://user-images.githubusercontent.com/21146741/116301383-fb0ae080-a76d-11eb-857b-32a53a630f53.png)

Reply:
Oh interesting.. I actually like having the comma because it makes it clear that the number refers to the icon on its left and not the one on its right.

Challenge: Insert the commas using CSS `::before` and `:last-child`.  :rocket:

Reply:
@jeremylenz : Had to do a CSS crash course.. 😄 

How does this look?

https://github.com/Katello/katello/compare/db0d25e085d584e9f01369729d6f2a3d74b5265e..325a83a94af9f7fe1af5160c60f39143c0503d59

![Screenshot from 2021-04-27 16-48-55](https://user-images.githubusercontent.com/21146741/116310806-783b5300-a778-11eb-8be9-a1c38cee033c.png)

Reply:
awesome :smile: 

This made the spacing a bit better for me:
```css
display: inline-flex;
margin-left: 2px;
margin-right: 5px;
```

Reply:
Added this.. 👍🏽